### DISCUSSION TITLE
AZTV7 and 7.5

### DISCUSSION THREAD

**Discussion by bnanewbie** [iptv-org/community#1010] [Q&A]
I am new to this and wondering if there is a known m3u for the sub channel of 7.5 out of Arizona?

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
